target/linux/ath79/dts/ar9132_buffalo_wzr-hp-g300nh.dtsi

```diff
@@ -0,0 +1,252 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9132.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "buffalo,wzr-hp-g300nh", "qca,ar9132";
+	model = "Buffalo WZR-HP-G300NH";
```

Comment:
Please move that to the DTS files and remove it here.
While using the -rb/-s syntax for compatible seems fine, you should provide a better explanation for the model, e.g.
`model = "Buffalo WZR-HP-G300NH (rtl8366rb)";`
and
`model = "Buffalo WZR-HP-G300NH (rtl8366s)";`

Reply:
done